### PR TITLE
Dynamic combobox

### PR DESCRIPTION
**Pull request by LAShemilt**
Did not need to do anything for the dynamic combobox in the params, as the methods were too bespoke to add here. However, I did need to add a Button type to the widget registery. 

Misslabelled pull request

### CODE REVIEW

File: rfi_file_monitor/utils/utils.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
+import gi
```
Comment (tschoonj) [inline rfi_file_monitor/utils/utils.py:1]: Not a fan of `utils.utils` to be honest... How about just renaming this file to `__init.py`?

File: rfi_file_monitor/utils/widgetparams.py [hunk 2]
```diff
@@ -70,6 +43,10 @@ def _spinbutton_value_changed_cb(self, spinbutton: Gtk.SpinButton, param_name: s
     def _combobox_changed_cb(self,combobox: Gtk.ComboBoxText, param_name: str):
         self._params[param_name] = combobox.get_active_text()
 
+    @final
+    def _button_clicked_cb(self, button: Gtk.Button, param_name:str):
```
Comment (tschoonj) [inline rfi_file_monitor/utils/widgetparams.py:47]: I don't see how you can use a button to back a param that can be used in the pipeline... this is for the dynamic combobox I guess?

**Review by tschoonj — CHANGES_REQUESTED** [review]

File: rfi_file_monitor/utils/widgetparams.py (hunk 2 above)
Comment (LAShemilt) [reply rfi_file_monitor/utils/widgetparams.py]: removed this now as it was not correct

File: rfi_file_monitor/utils/utils.py (hunk 1 above)
Comment (LAShemilt) [reply rfi_file_monitor/utils/utils.py]: been changed

**Review by tschoonj — APPROVED** [review]

**Pull request merged**